benchmarks/definition/config.py

```diff
@@ -0,0 +1,33 @@
+"""This module contains the configuration of a benchmark scenario."""
+
+from typing import TypeVar
+
+from attrs import define, field
+from attrs.validators import deep_mapping, instance_of
+
+from baybe.recommenders.base import RecommenderProtocol
+from baybe.serialization.mixin import SerialMixin
+
+BenchmarkConfig = TypeVar("BenchmarkConfig", bound="SerialMixin")
+
+
+@define(frozen=True)
+class RecommenderConvergenceAnalysis(SerialMixin):
+    """The configuration for the benchmark settings."""
+
+    batch_size: int = field(validator=instance_of(int))
+    """The batch size for the benchmark."""
+
+    n_doe_iterations: int = field(validator=instance_of(int))
+    """The number of Design of Experiment iterations."""
+
+    n_mc_iterations: int = field(validator=instance_of(int))
```

Comment:
@AVHopp @Scienfitz: an opportunity to rethink the naming of these arguments (I noticed we've never really talked about that). What do you think about the current ones, also in the simulation module? Is "iterations" actually a good word for it?

Reply:
do you see a problem with these names or think they are not easily understandable? if no -> no discussion needed

Reply:
OK, if you don't see a problem, then all good 👍🏼. Let me close